tests/conftest.py

```diff
@@ -1615,8 +1615,8 @@ def duts_running_config_facts(duthosts):
     Returns:
         dict: {
             <dut hostname>: [
-                {asic0_cfg_facts},
-                {asic1_cfg_facts}
+                (asic0_idx, {asic0_cfg_facts}),
+                (asic1_idx, {asic1_cfg_facts})
```

Comment:
Why was tuple chosen here instead of using another dict with asic index as the key?
Then the places where this cfg_fact is accessed could have mostly remained the same as it is accessing a list with list index.

        dict: {
            <dut hostname>: {
                <asic idx>: {asic0_cfg_facts},
                }
              }

Reply:
@SuvarnaMeenakshi I selected `tuple` considering it will be easy to port the change. With Just extra indexing. I think if we had dict chnages would be more as to get correct index as key  to access dictionary value